templates/apps/sumo-aws-apps/cloudtrail/cloudtrail.template.yaml

```diff
@@ -343,6 +349,24 @@ Resources:
           - SumoLogicHelperRole
           - Arn
 
+  ValidationCloudTrailBucket:
+    Condition: create_validation_logs_s3_bucket
+    Type: Custom::AWSValidation
+    Properties:
+      ServiceToken: !GetAtt SumoLogicHelperFunction.Arn
+      PARAMETER_NAME: "CloudTrailLogsBucketName"
+      PARAMETER_VALUE: !Ref CloudTrailLogsBucketName
+      PARAMETER_ALLOWED_PATTERN: '(?!(^xn--|.+-s3alias$))^[a-z0-9][a-z0-9-]{1,61}[a-z0-9]$'
+      
+  ValidationCloudTrailPathExpression:
+    Condition: cloudtrail_validation_path_expression
+    Type: Custom::AWSValidation
+    Properties:
+      ServiceToken: !GetAtt SumoLogicHelperFunction.Arn
+      PARAMETER_NAME: "cloudtrail bucket path expression (CloudTrailBucketPathExpression)"
+      PARAMETER_VALUE: !Ref CloudTrailBucketPathExpression
```

Comment:
we should also changed default value for CloudTrailBucketPathExpression to `AWSLogs/*` the simple `*` seems risky

Reply:
fixed